### PR TITLE
chore: Update help text

### PR DESCRIPTION
**Pull request by stujfiter**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -250,3 +250,4 @@ A special thanks to everyone that has contributed to seCureLI so far:
 - Jeff Schumacher
 - Caleb Tonn
 - Josh Werner
+- Korey Earl
```
Comment (calebtonn) [inline CONTRIBUTING.md:253]: We are doing this alphabetical order by last name.

File: secureli/resources/files/epilog.md [hunk 2]
```diff
@@ -11,6 +11,6 @@ SeCureLI currently supports the following languages:
 {% endfor %}
 
 
-For more information, check us out on Confluence at {{ confluence_url }}
+For more information, check us out on the GitHub wiki at {{ confluence_url }}
```
Comment (calebtonn) [inline secureli/resources/files/epilog.md:14]: variable would match whatever you decide it would be in setup.py

File: CONTRIBUTING.md (hunk 1 above)
Comment (calebtonn) [inline CONTRIBUTING.md:253]: We are doing alphabetical list by last name.

File: secureli/resources/files/epilog.md (hunk 2 above)
Comment (stujfiter) [reply secureli/resources/files/epilog.md:14]: I'm not sure what you mean by this.  Can you elaborate?

**Review by calebtonn — APPROVED** [review]

**Pull request merged**